db/migrate/20230202173641_create_form_templates.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateFormTemplates < ActiveRecord::Migration[7.0]
+  def change
+    create_table :form_templates do |t|
+      t.string :name, null: false
+      t.text :template, null: false
```

Comment:
We still need to enforce limits at db level here, see our lore topic

Reply:
Ah yes, forgot about that. Just pushed a commit.